src/plugins/node-workspace.ts

```diff
@@ -271,20 +278,31 @@ export class NodeWorkspace extends WorkspacePlugin<Package> {
     );
     const packageJson = updatedPackage.toJSON() as PackageJson;
     const version = Version.parse(packageJson.version);
+
+    const strategy = this.strategiesByPath[updatedPackage.location];
+    const latestRelease = this.releasesByPath[updatedPackage.location];
+
+    const basePullRequest = strategy
```

Comment:
Make a base pull request that contains the "extra-files" updaters as well as the changelog header.